loadtest.go

```diff
@@ -328,20 +328,22 @@ func (l *loadTest) runPublisher(ctx context.Context, client Client, userNum int6
 							Time:    timeNow(),
 						},
 					})
-					l.log.Debug("publishing message", "channel", channel, "size", len(data))
-					err := client.Publish(ctx, channel, []*ably.Message{{
-						Data:   data,
-						Extras: extras,
-					}})
-					if errors.Is(err, context.Canceled) {
-						l.log.Debug("publisher stopped", "channel", channel)
+					errG.Go(func() error {
```

Comment:
Given the passed function never returns an error, should this just be a simple `go func() { ... }`?

Reply:
I used an `errgroup` because it is `Wait()`ed on termination. Using a goroutine would result in a leak I think.

Reply:
Ah ok 👍